### PR TITLE
Add HoTT to Windows Installer (8.10)

### PR DESCRIPTION
**Pull request by MSoegtropIMC**
On request of a teacher I added HoTT to the Windows installer.

I made an upstream PR for the HoTT patch, but there are issues with HoTT's CI, so I would first merge it with the patch and later remove it. See (https://github.com/HoTT/HoTT/pull/1058)

### CODE REVIEW

File: dev/ci/ci-basic-overlay.sh [hunk 1]
```diff
@@ -69,7 +69,7 @@
 ########################################################################
 # HoTT
 ########################################################################
-: "${HoTT_CI_REF:=ed714382f980d54c055890f95bef0de4ff6bdc78}"
+: "${HoTT_CI_REF:=95aec99612b12747555534e444a0d7274dbcaaf4}"
```
Comment (Zimmi48) [inline dev/ci/ci-basic-overlay.sh]: Have you tested that this version is still compatible with Coq 8.10?

Comment (MSoegtropIMC) [reply dev/ci/ci-basic-overlay.sh]: Yes, it builds and I also did some additional tests in CoqIDE, so I have no reason to assume that this version is worse than another version. I will get additional feedback from people giving lectures on the HoTT summer school soon.

Please note that on the 810 branch full Windows installers are enabled now.

**Comment by MSoegtropIMC** [comment]
This has some unexpected effects and shouldn't be merged yet.
See `https://github.com/HoTT/HoTT/issues/1064`

**Comment by ejgallego** [comment]
This seems to have gone stale, I'm closing this in favor of #10589 which targets master. Once in master, it could be backported to 8.11.

**Pull request closed without merging**